RUFAS/biophysical/manure/parlor_cleaning.py

```diff
@@ -0,0 +1,119 @@
+        super().__init__(name, is_housing_emissions_calculator, config)
+
+    def receive_manure(self, manure_stream: ManureStream) -> None:
+        """
+        Receiving manure stream.
+
+        Parameters
+        ----------
+        manure_stream : ManureStream
+            The ManureStream instance being received.
+
+        """
+        super().receive_manure(manure_stream)
+        if self.manure_stream is None:
+            self.manure_stream = manure_stream
+        else:
+            self.manure_stream += manure_stream
+
+    def process_manure(self, conditions: CurrentDayConditions, time: Time) -> dict[str, ManureStream]:
+        """
+        Executes the daily manure processing operations.
+
+        Parameters
+        ----------
+        conditions : CurrentDayConditions
+            Current weather and environmental conditions that manure is being processed in.
+        time : Time
+            Time instance containing the simulations temporal information.
+
+        Returns
+        -------
+        dict[str, ManureStream]
+            Mapping between classification of manure coming out of this processor to the ManureStream containing the
+            manure information. If the processor is a separator, the classifications are "solid" and "liquid". Otherwise
+            the only classification is "manure".
+
+        """
+        if self.manure_stream is None or self.manure_stream.pen_manure_data is None:
+            info_map = {"class": self.__class__.__name__, "function": self.process_manure.__name__}
+            self._om.add_error(
+                "None type ManureStream.",
+                "The processed ManureStream or pen data of the manure stream is None type.",
+                info_map,
+            )
+            raise TypeError("TypeError: Handler tries to process 'NoneType' object ManureStream.")
+        num_animals = self.manure_stream.pen_manure_data.num_animals
+        self.fresh_water_volume_used_for_milking = self.determine_fresh_water_volume_used_for_milking(num_animals)
+        return super().process_manure(conditions, time)
+
+    def determine_cleaning_water_volume_in_main_barn(
+        self, num_animals: int, cleaning_water_use_rate: float, cleaning_water_recycle_fraction: float
+    ) -> float:
+        """
+        Calculates the volume of fresh (non-recycled) cleaning water used for, and ultimately added to, a single manure
+         stream on a single simulation day by the manure handler.
+
+        Parameters
+        ----------
+        num_animals : int
+            Number of animals.
+        cleaning_water_use_rate : float
+            The use rate of cleaning water (unitless).
+        cleaning_water_recycle_fraction : float
+            The fraction of cleaning water recycled (unitless).
+
+        Returns
+        -------
+        float
+            The volume of fresh (non-recycled) cleaning water (m^3).
```

Comment:
```suggestion
            The volume of fresh (non-recycled) cleaning water added to the manure stream (m^3).
```
Not sure if this is correct but think it would be helpful to be specific.